src/lib/rest/Verb.cpp

```diff
@@ -48,3 +48,24 @@ const char* verbName(Verb verb)
 
   return "Unknown verb";
 }
+
+
+
+/* ****************************************************************************
+*
+* stringToVerb - 
```

Comment:
Maybe a reference to the HTTP RFC from which this has been taken could be a good idea.